docs/sdk/api-reference.md

```diff
@@ -1389,20 +1199,10 @@ Initializes an Input object.
 - `root_certificates_path` (*str*) - Path to SSL root certificates file
 - `**kwargs` - Additional keyword arguments
 
-### Inputs._get_proto()
+### Inputs._get_proto
```

Comment:
I think this is also a internal method. WDYT @sanjaychelliah

Reply:
Yes this should be removed

Reply:
As this is an internal method i have removed this from the doc